docs/src/main/sphinx/sql/set-path.rst

```diff
@@ -0,0 +1,51 @@
+========
+SET PATH
+========
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET PATH path-element, path-element, ...
+
+Description
+-----------
+
+Update the session to define a collections of paths to a functions or table
+functions in a catalog. Each path-element uses a period-separated syntax of to
+specify the catalog name and schema location ``<catalog>.<schema>``of the
+function or only the schema location ``<schema>`` in the current catalog. The
+current catalog is set with :doc:`use` or as part of a client tool connection.
+Catalog and schema must exist. Function names must be unique across all path
+elements.
+
+Examples
+--------
+
+The following example sets a path to access functions in the ``system`` schema
+of the ``example`` catalog::
+
+    SET PATH example.system;
+
+The catalog uses the PostgreSQL connector, and you can therefore use the
+:ref:`query table function <postgresql-query-function>` directly, without the
+full catalog and schema access path::
+
+    SELECT
```

Comment:
I have not tested this.. do you know if this works @kasiafi or is supposed to work.

Reply:
Yes, this should work.